portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,23 @@ function addRandomFact() {
   // Add it to the page.
   document.getElementById('random-facts-container').innerText = randomFact;
 }
+
+function displayMessages() {
+  fetch('/data').then(response => response.json()).then((messageArray) => {
+    
+    const messageList = document.getElementById('message-container');
+    messageList.innerHTML = '';
```

Comment:
Please use a for-loop based on the size of the array

Reply:
Done!!